packages/studio-plugin/src/sourcefiles/ComponentFile.ts

```diff
@@ -62,8 +63,9 @@ export default class ComponentFile {
 function getCssFilesFromDependencyTree(dependencyTree: Tree): string[] {
   const cssFiles = Object.entries(dependencyTree).reduce(
     (cssFiles, [filename, subDependencyTree]) => {
-      if (filename.includes(".css")) {
-        cssFiles.add(filename);
+      const unixFilepath = upath.toUnix(filename);
+      if (unixFilepath.includes(".css")) {
```

Comment:
should this check that it ends with `.css` rather than includes it?

Reply:
yes!